prepare for 1.11 release

webern opened the pull request:
Prep the changelog and release.toml for 1.11 release, anticipated for next week (date could change).

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

webern committed:
prepare for 1.11 release

stmcginnis commented:
We should probably mention https://github.com/bottlerocket-os/bottlerocket/pull/2377

webern commented:
> We should probably mention #2377

It was intended to be there, I will fix it.

stmcginnis reviewed (approved):
This looks good as far as the things I have been involved with.

zmrow commented on CHANGELOG.md:
```diff
@@ -1,3 +1,82 @@
+
+## OS Changes
+
+* Prevent a panic early-boot-config when there is no IMDS Region ([#2493])
```
Should probably be:
```
Prevent a panic in `early-boot-config`...
```

zmrow commented on CHANGELOG.md:
```diff
@@ -1,3 +1,82 @@
+* Update EKS-D to 1.23-6 for k8s-1.23 variants ([#2488])
+* Update EKS-D to 1.22-11 for k8s-1.22 variants ([#2490])
+* Add a timeout when calling EKS for configuration values ([#2566])
+* Enable IAM Roles Anywhere with the k8s `ecr-credential-provider plugin ([#2377], [#2553])
```
I think we're missing a backtick after `ecr-credential-provider`

cbgbt commented on CHANGELOG.md:
```diff
@@ -1,3 +1,82 @@
+* Add support for publishing to AWS organizations ([#2484])
+* Remove unnecessary dependencies when building grub ([#2495])
+* Switch to the latest Dockerfile frontend for builds ([#2496])
+* Prepare foundations for Secure Boot and image resigning ([#2505])
```
Nit: I think "re-signing" might be more correct here

webern replied:
lol, I'm glad it doesn't mean the PR author is resigning.

webern commented:
Addresses comments so far
https://github.com/bottlerocket-os/bottlerocket/compare/0a1dabf7b540b40cf83dd91969a91d18295a511d..c3d3f3822d7194ee68809ddbb9bda445ade7e73e

webern commented:
^ Add PRs that will merge soon

https://github.com/bottlerocket-os/bottlerocket/compare/c3d3f3822d7194ee68809ddbb9bda445ade7e73e..3082d37d44b1983995355e694e0148ef2faa36ac

webern commented:
^ Add 2 more PRs

https://github.com/bottlerocket-os/bottlerocket/compare/3082d37d44b1983995355e694e0148ef2faa36ac..a59901dfd0636a33c932e6f78a63527df6a722b5

bcressey commented on CHANGELOG.md:
```diff
@@ -1,3 +1,91 @@
+
+## OS Changes
+
+* Prevent a panic `early-boot-config` when there is no IMDS Region ([#2493])
```
I'd probably group this with `AWS` changes.

```suggestion
* Prevent a panic in `early-boot-config` when there is no IMDS region ([#2493])
```

bcressey commented on CHANGELOG.md:
```diff
@@ -1,3 +1,91 @@
+
+* Prevent a panic `early-boot-config` when there is no IMDS Region ([#2493])
+* Update grub to 2.06-42 ([#2503])
+* Allow interface configuration via Ethernet permanent address ([#2519])
```
This will likely be read as `net.toml` supporting this feature, which it doesn't - it's just a patch we added to wicked to prepare for that functionality.

```suggestion
* Bring back wicked support for matching interfaces via hardware address ([#2519])
```

bcressey commented on CHANGELOG.md:
```diff
@@ -1,3 +1,91 @@
+* Update grub to 2.06-42 ([#2503])
+* Allow interface configuration via Ethernet permanent address ([#2519])
+* Allow bootstrap containers to manage swap ([#2537])
+* Add systemd analyze commands to troubleshooting log collection tool ([#2550])
```
```suggestion
* Add `systemd-analyze` commands to troubleshooting log collection tool ([#2550])
```

bcressey commented on CHANGELOG.md:
```diff
@@ -1,3 +1,91 @@
+* Serialize bootconfig values correctly when the value is empty ([#2565])
+* Update zlib, libexpat, libdbus, docker-cli ([#2583])
+* Update host containers ([#2574])
+* Unmask /sys/firmware from host containers in host-ctr ([#2573])
```
```suggestion
* Unmask /sys/firmware from host containers ([#2573])
```

bcressey commented on CHANGELOG.md:
```diff
@@ -1,3 +1,91 @@
+
+### ECS
+
+* Add additional ECS API configurations ([#2527])
```
It would be nice to say what was added - maybe multiple bullet points mapping to the same PR.

bcressey commented on CHANGELOG.md:
```diff
@@ -1,3 +1,91 @@
+
+### Kubernetes
+
+* Update EKS-D to 1.23-6 for k8s-1.23 variants ([#2488])
+* Update EKS-D to 1.22-11 for k8s-1.22 variants ([#2490])
+* Add a timeout when calling EKS for configuration values ([#2566])
+* Enable IAM Roles Anywhere with the k8s `ecr-credential-provider` plugin ([#2377], [#2553])
+* Kubernetes updates ([#2578], [#2580], [#2582]),
```
Let's combine all the Kubernetes updates for each version and indicate the "final" version that was shipped in the release note, even if we link to all the PRs. Right now the notes suggest that `1.23-6` and `1.22-11` are in the release, which is incorrect.

webern commented:
^ address @bcressey 's comments

https://github.com/bottlerocket-os/bottlerocket/compare/a59901dfd0636a33c932e6f78a63527df6a722b5..7bc5f5391006090c76495b7c07a198001b818326

webern commented:
^ fix a whitespace typo

https://github.com/bottlerocket-os/bottlerocket/compare/7bc5f5391006090c76495b7c07a198001b818326..4aedfd5259253397248b20808c679375cdb56df1

zmrow reviewed (approved):
:volleyball:
